modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java

```diff
@@ -296,7 +296,8 @@ private JSONArray _getTrafficSourcesJSONArray() {
 					).findFirst(
 					).map(
 						trafficSource -> trafficSource.toJSONObject(
-							helpMessageMap.get(name), title)
+							helpMessageMap.get(name), _themeDisplay.getLocale(),
```

Comment:
Are we sure we want the user language? Or do we prefer the site default language or something like that?

Reply:
we need the current user locale, in order to display the country name using this language

Reply:
But are we retrieving the keywords in the user language?